YesGraph/yesgraph-superwidget
# Analytics manager

#### What’s this PR do?
- Fixes a bug whereby user data wasn't getting passed through properly
- Adds more analytics events, managed by an `AnalyticsManager` object.

#### Any background context you want to provide?
The bug around user data was discovered when Ivan saw that the invite links weren't getting rendered properly. We can and should add more tests around this (and other things).

#### Where should the reviewer start?
The new `AnalyticsManager` constructor in src/dev/yesgraph.js

#### How should this be manually tested?
- Run `npm test`
- Open up src/demo_widget.html. (If the CSS doesn't load, run `gulp` from the command line.) In the console, you should see a successful hit to the sdk analytics endpoint:
 <img width="633" alt="screen shot 2016-06-15 at 2 08 16 pm" src="https://cloud.githubusercontent.com/assets/10701968/16097556/2e97664c-3303-11e6-9a58-2ae1a087a19f.png">

#### What are the relevant tickets?
User data bug: https://app.asana.com/0/59202558034519/144828561826343
Analytics: https://app.asana.com/0/59202558034519/136838727888654
